Sources/SourceKitLSPAPI/PluginTargetBuildDescription.swift

```diff
@@ -41,6 +41,18 @@ struct PluginTargetBuildDescription: BuildTarget {
 
     var headers: [URL] { [] }
 
+    var resources: [URL] {
+        return target.underlying.resources.map { URL(fileURLWithPath: $0.path.pathString) }
+    }
+
+    var ignored: [URL] {
+        return target.underlying.ignored.map { URL(fileURLWithPath: $0.pathString)}
```

Comment:
nit: formatting is consistent
```suggestion
        return target.underlying.ignored.map { URL(fileURLWithPath: $0.pathString) }
```

Reply:
Fixed